### DISCUSSION TITLE
Welcome to heyu Discussions!

### DISCUSSION THREAD

**Discussion by jkrzyszt** [HeyuX10Automation/heyu#29] [General]
## 👋 Welcome to Heyu Discussions!
  Since we, Heyu User Community, lost our ability to communicate over the heyu_users Yahoo group, maybe Github Discussions can fill the gap.  I have no idea how to reach our members to let them know about this possibility, but I hope they will discover Discussions while visiting our Git repository and join.  New users are also welcome.  So, using words suggested by Github for an introductory message:
  * Ask questions on Heyu you’re wondering about.
  * Share Heyu related ideas.
  * Engage with other community members.
  * Welcome others. 
  * Remember that this is a community we build together.

  To get started, you are welcome to comment below with an introduction of yourself and tell us about what you do with Heyu

### Reporting issues:
* Please try to use the Issues tab only for reporting Heyu software bugs.  If not sure then please open a new topic under Discussions, we can always create a new issue from it if it occurs a software bug.  Please discuss other issues, which are not software bugs, under Discussions.

* When reporting a new issue, whether software bug or other, please try to avoid posting them as replies to unrelated topics -- just open a new one.

* Your help is crucial for successful resolution of bugs.  If a fix is available for testing, please give it a try and report your results.

### Bringing maintainer's attention:
Github notifications on new issues reported, supposed to be sent to me via email, don't work reliably for me.  To make sure I'm notified, please feel free to mention me in your report, like this: @jkrzyszt.

Thanks,
Janusz

**Comment by arnieSkyNet** [comment]
heyu is still the core of my home automation, now running on a pi3 with bwssytems/ha-bridge ( https://github.com/bwssytems/ha-bridge ) which allows me to put all the heyu commands into it and then my Alexa upon discovering allows me to control anything heyu can do.

As you all know heyu scripts go beyond X10 control, in my case one of these is irtrans ( http://www.irtrans.com/en/index.php ) doing all my infra Red stuff, like:

* Alexa tune on my iTunes 
Alexa->habridge->heyu->X10 on Windows pc

* Alexa tune on ITV
Alexa->habridge->heyu->irtrans send 111 which switches tv to itv

Sometimes habridge runs code directly that’s nothing to do with heyu but usually that means I add it into heyu scripts so when away from home I can still do this without Alexa via X10 Commander App.


Alexa opens my door so is switched off when no one at home (via X10 module)

**Comment by hoclun-rigsep** [comment]
Thanks for the welcome, just another user who wants to add to the evidence that this code is still in use and still valuable. 
Thanks devs.

**Comment by pleurastic** [comment]
I finally figured out where the group went...I think.  My ASUS eeepc is still running heyu for me and provides some lighting controls, a couple of access door security switches and the garage door opening alarm.  It's so trouble free that I forget about it.  I should probably move it to a Raspberry Pi but I haven't taken the time to do so, yet.

**Comment by mav1025** [comment]
I've been using x10 for many years and with commands through google assistant (IFTTT + some apps + Windows - Firecracker x10 module) Looking at running x10 on Raspberry Pi. I am also another user that finds this code valuable.. THANK YOU!

**Comment by three-jeeps** [comment]
HELP....Hope someone reads this.....since the yahoo group went away don't know where to post questions......
Been using heyu with x10 since V2.7 on an Asus EEEPC with Ubuntu 16.04, heyu ver 2.10.
Been running for years with no problems, till recently. Noticed my heyu logfile not being updated.  I rebooted the machine and upon starting heyu, I get the dreaded error:
 Can't open tty line.  Check the permissions.

In the distant past with same error, I remember changing permissions on the serial line fixed the problem...specifically:
sudo chmod 777 /dev/ttyS0, (or was it chmod 666 /dev/ttyS0, dont remember exactly)

Unfortunately this did not fix the problem......then again, I realized that the CMA11 is connected via a USB port with a USB to serial converter cable (the same one I've been using for the last 4-5 years....) My 'fix' was for a prior machine that had a real serial line on it.....

So I am stumped on how to fix this issue....help please?
J
